ios/FluentUI/Vnext/Persona/PersonaView.swift

```diff
@@ -14,8 +14,8 @@ import SwiftUI
 /// `onTapAction` provides tap gesture for PersonaView.
 ///
 @objc public protocol MSFPersonaViewState: MSFAvatarState {
-    var titleTrailingAccessoryView: UIView? { get set }
-    var subtitleTrailingAccessoryView: UIView? { get set }
+    var titleTrailingAccessoryUIView: UIView? { get set }
```

Comment:
We need to add the AnyView properties to the protocol as well. (Considering the Swift/SwiftUI only scenario).

Reply:
The AnyView properties are not compatible with the objc protocol so I made another child protocol.